pkg/docker/entrypoint.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/sh
+set -e
+
+# This docker image supports sending either a flag or a command as the docker
+# command. When a flag is sent, it will be passed on to the metadata agent 
+# process. Anything else will be interpreted as the command to be run.
+#
+# Passing a flag.
+# $ docker run -it {image:tag} -v
+#
+# Passing a command.
+# $ docker run -it {image:tag} /bin/bash
+#
+# Default behavior uses CMD defined in Dockerfile.
+# $ docker run -it {image:tag}
```

Comment:
This comment is not related to the following `if`, so let's add a blank line after it.

Reply:
Done.